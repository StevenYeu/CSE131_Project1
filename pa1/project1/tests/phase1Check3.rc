function : void foo() {
   int x;
   x  = 5.5 + true;
   int y;
   y = 1.1;
}
