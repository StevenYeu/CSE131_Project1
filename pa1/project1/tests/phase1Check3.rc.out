Error, "/tmp/cs131_testrunner_RQ0qMF/file.rc": 
  Incompatible type bool to binary operator +, numeric expected.
Error, "/tmp/cs131_testrunner_RQ0qMF/file.rc": 
  Value of type float not assignable to variable of type int.
Compile: failure.
